docs/databases/mongodb/mongodb-example-shortguide/index.md

```diff
@@ -0,0 +1,90 @@
+---
+author:
+  name: Linode
+  email: docs@linode.com
+description: 'Shortguide that displays usage example for MongoDB.'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-03-11
+modified_by:
+  name: Heather Zoppetti
+published: 2020-03-11
+title: MongoDB Example
+keywords: []
+headless: true
+show_on_rss_feed: false
+---
+
+Much of MongoDB's popularity comes from its ease of integration. Interactions with databases are done via JavaScript methods, but [drivers for other languages](https://docs.mongodb.com/ecosystem/drivers/) are available. This section will demonstrate a few basic features, but we encourage you to do further research based on your specific use case.
+
+1.  Open the MongoDB shell using the `example-user` we created above:
+
+        mongo -u example-user -p --authenticationDatabase user-data
+
+1.  Create a new database. This example calls it `exampleDB`:
+
+        use exampleDB
+
+    Make sure that this database name corresponds with the one for which the user has read and write permissions (we added these permissions in Step 7 of the previous section).
```

Comment:
I think the step is actually step 6.

Reply:
Ok - I had to take out the step number because this is different in each guide it's used inside